src/kem.rs

```diff
@@ -31,709 +31,32 @@
 //!
 //! [FIPS 203]: https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.203.ipd.pdf
```

Comment:
Not you, but this comment makes no sense. Can you add a short comment that this is a KEM API?